packages/plugin-eslint/src/lib/runner.integration.test.ts

```diff
@@ -2,26 +2,36 @@ import os from 'node:os';
 import path from 'node:path';
 import { fileURLToPath } from 'node:url';
 import { type MockInstance, describe, expect, it } from 'vitest';
-import type { AuditOutput, AuditOutputs, Issue } from '@code-pushup/models';
+import type {
+  AuditOutput,
+  AuditOutputs,
+  Issue,
+  RunnerFilesPaths,
+} from '@code-pushup/models';
 import { osAgnosticAuditOutputs } from '@code-pushup/test-utils';
 import { readJsonFile } from '@code-pushup/utils';
 import type { ESLintTarget } from './config.js';
 import { listAuditsAndGroups } from './meta/index.js';
-import {
-  RUNNER_OUTPUT_PATH,
-  createRunnerConfig,
-  executeRunner,
-} from './runner/index.js';
+import { createRunnerConfig, executeRunner } from './runner/index.js';
 
 describe('executeRunner', () => {
   let cwdSpy: MockInstance<[], string>;
   let platformSpy: MockInstance<[], NodeJS.Platform>;
 
-  const createPluginConfig = async (eslintrc: ESLintTarget['eslintrc']) => {
+  const createPluginConfig = async (
+    eslintrc: ESLintTarget['eslintrc'],
+  ): Promise<RunnerFilesPaths> => {
     const patterns = ['src/**/*.js', 'src/**/*.jsx'];
     const targets: ESLintTarget[] = [{ eslintrc, patterns }];
     const { audits } = await listAuditsAndGroups(targets);
-    await createRunnerConfig('bin.js', audits, targets);
+    const { outputFile } = await createRunnerConfig('bin.js', audits, targets);
+    return {
+      runnerOutputPath: outputFile,
+      runnerConfigPath: outputFile.replace(
+        'runner-output.json',
+        'plugin-config.json',
+      ),
```

Comment:
Maybe we could add an optional `configFile` to the runner config schema? It would save you from doing this output file replacement in several tests. And the core could then also take care of deleting the config file as well :thinking: